modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -2262,7 +2262,6 @@ private void onCacheStarted(GridCacheContext cacheCtx) throws IgniteCheckedExcep
                 ", mode=" + cfg.getCacheMode() +
                 ", atomicity=" + cfg.getAtomicityMode() +
                 ", backups=" + cfg.getBackups() +
-                ", mvcc=" + cacheCtx.mvccEnabled() +
```

Comment:
Do we need keep mvcc=false untill MVCC is entirely removed?

Reply:
No, we're removing this, step by step :)